### PR TITLE
feat: client SDK - add explore module 

### PR DESCRIPTION
**Pull request by paulxuca**

### CODE REVIEW

File: packages/client/src/explore/Explore.spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+      await expect(
+        explore.explorePublications({
+          sortCriteria: PublicationSortCriteria.TopCommented,
+        }),
+      ).resolves.not.toThrow();
+    });
+  });
+
+  describe(`when the instance is authenticated and method ${Explore.prototype.exploreProfiles.name} is called`, () => {
```
Comment (krzysu) [inline packages/client/src/explore/Explore.spec.ts:22]: there is sth wrong with the descriptions here and above 😄

File: packages/client/src/explore/graphql/explore.graphql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        ...Comment
+      }
+    }
+    pageInfo {
+      ...CommonPaginatedResultInfo
+    }
+  }
+}
+
+query ExploreProfiles($observerId: ProfileId, $limit: LimitScalar!, $cursor: Cursor) {
```
Comment (krzysu) [inline packages/client/src/explore/graphql/explore.graphql:22]: I think in this low level SDK we should just expose all what API accepts, so `ExploreProfilesRequest` in this case

File: packages/client/src/explore/Explore.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    const client = new GraphQLClient(config.environment.gqlEndpoint);
+
+    this.sdk = getSdk(client);
+  }
+
+  async explorePublications(request: ExplorePublicationRequest) {
+    return this.sdk.ExplorePublications({ request });
+  }
+
+  async exploreProfiles(request: ExploreProfilesRequest) {
```
Comment (krzysu) [inline packages/client/src/explore/Explore.ts:21]: think about pagination helper like we have `PaginatedReadResult` in react package

Comment (paulxuca) [reply packages/client/src/explore/Explore.ts:21]: Adding it as a separate PR to explore first 👍

**Review by krzysu — APPROVED** [review]
LGTM!

**Pull request merged**